packages/model-viewer/examples/customizing-ui.html

```diff
@@ -74,6 +74,25 @@ <h4></h4>
     </div>
   </div>
 
+  <div class="sample" style="background: linear-gradient(#ffffff, #ada996);">
+    <div id="demo-container-2" class="demo"></div>
+    <div class="content">
+      <div class="wrapper">
+        <div class="heading">
+          <h2 class="demo-title">Transparent Background</h2>
+          <h4></h4>
+        </div>
+        <example-snippet stamp-to="demo-container-2" highlight-as="html">
+          <template>
+            <model-viewer ar camera-controls ios-src="assets/Astronaut.usdz" src="assets/Astronaut.glb" background-color="transparent" alt="A 3D model of an astronaut">
```

Comment:
nit: Everything inside the `<template>` should be indented starting over from the left. If you look at this page in your browser you'll see why. Also, can we add an image or some text or something behind this so that it's clear the transparency is working? These dev pages serve as manual regression tests for this project.

Reply:
Ah, I just saw the linear gradient; the trouble is that it's not inside the `<template>`, so it's not a visible part of the demo code.

Reply:
I modified the example to integrate the gradient and a text in the template. Thanks for all the feedback.